examples/v0/account_management/create_customer.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This example illustrates how to create a new customer under a given
+manager account.
+
+Note by default this account will only be accessible via its parent Google Ads
+manager account.
+"""
+
+from __future__ import absolute_import
+
+import argparse
+import six
+import sys
+
+import google.ads.google_ads.client
+
+
+def main(client, customer_id):
+    customer_service = client.get_service('CustomerService')
+    resource_name = customer_service.customer_path(customer_id)
+    customer = client.get_type('Customer')
+    customer.resource_name = resource_name
+    customer.descriptive_name.value = 'A new customer.'
```

Comment:
Although the descriptive name does not need to be unique, let's use the [same approach we used with the SOAP example](https://github.com/googleads/googleads-python-lib/blob/master/examples/adwords/v201809/account_management/create_account.py#L44) so users can easily find the new account.

```
today = datetime.today().strftime('%Y%m%d %H:%M:%S')
customer.descriptive_name.value = 'Account created with CustomerService on %s' % today
```

Reply:
Done